tests/unit/phase/map.py

```diff
@@ -75,3 +77,61 @@ def test_map_empty_geometry_column():
         "entry-date,geometry,legislation,name,notes,organisation,point,prefix,"
         "reference,start-date\r\n,,,,,,MULTIPOLYGON(),,,,,,,,\r\n"
     )
+
+
```

Comment:
these tests still need moving to the appropriate location in the test directory. Just because the file exists does not mean that we shouldn't focus on improving the tests directory structure.

Reply:
I thought it would be a separate ticket - so as not to bind errors related to moving, to this specific task.